docs/docs.go

```diff
@@ -289,12 +167,12 @@ type swaggerInfo struct {
 
 // SwaggerInfo holds exported Swagger Info so clients can modify it
 var SwaggerInfo = swaggerInfo{
-	Version:     "1.0",
-	Host:        "https://benefit-service-dev.dev.dts-stn.com",
-	BasePath:    "/",
+	Version:     "",
+	Host:        "",
+	BasePath:    "",
 	Schemes:     []string{},
-	Title:       "Benefit Service",
-	Description: "This service returns information about Benefits",
+	Title:       "",
+	Description: "",
```

Comment:
Anyone know why versions are being stripped out when I generate the docs? Is there a special command?